src/app/shared/common_function.spec.ts

```diff
@@ -189,4 +189,14 @@ describe('common functions', () => {
     expect(generateEbiOntologyLink('ncbitaxon', 'http://purl.obolibrary.org/obo/NCBITaxon_9823')).
     toEqual('https://www.ebi.ac.uk/ols/ontologies/ncbitaxon/terms?iri=http://purl.obolibrary.org/obo/NCBITaxon_9823');
   });
+
+  it('getProtocolLink should return correct link when uses old firebase server', () => {
+    expect(getProtocolLink('https://ftp.faang.ebi.ac.uk/test/test.pdf'))
+      .toEqual('https://hx.fire.sdo.ebi.ac.uk/fire/public/faang/test/test.pdf');
+  });
+
+  it('getProtocolLink should return the same link when does not use old fire server', () => {
+    expect(getProtocolLink('https://test.com/test/test.pdf'))
+      .toEqual('https://test.com/test/test.pdf');
+  });
```

Comment:
the old test case included converting ftp to http, missing in the new test case.

Reply:
Done

Reply:
please add another test case
getProtocolLink('ftp://ftp.faang.ebi.ac.uk/test/test.pdf')

after that it is ready to go

Reply:
Done